float64 timestamp
tugboat_control/BoatStatus[] boat